plugin-perf/README.md

```diff
@@ -1,3 +1,85 @@
 # Perf plugin
 
-Profiles processes and cgroups with Linux `perf_events` feature, similar to the `perf` command-line tool.
+The perf plugin creates an Alumet **source** that collects measurements using Linux `perf_events` feature, which profiles processes an cgroups, in a similar way as the `perf` command-line tool.
```

Comment:
Actually, `perf_events` can also observe system-level metrics. Perhaps you can take inspiration from the doc of perf:
> Performance counters are CPU hardware registers that count hardware events such as instructions executed, cache-misses suffered, or branches mispredicted. They form a basis for profiling applications to trace dynamic control flow and identify hotspots.